idd/Energy+.idd.in

```diff
@@ -7361,62 +7364,26 @@ MaterialProperty:PhaseChange,
-       \type real
-  N25,  \field Enthalpy 12
-       \note for Temperature-enthalpy function corresponding to temperature 14
-       \units J/kg
-       \type real
-  N26, \field Temperature 13
-       \note for Temperature-enthalpy function
-       \units C
-       \type real
-  N27, \field Enthalpy 13
-       \note for Temperature-enthalpy function corresponding to temperature 15
-       \units J/kg
-       \type real
-  N28, \field Temperature 14
-       \note for Temperature-enthalpy function
-       \units C
-       \type real
-  N29, \field Enthalpy 14
-       \note for Temperature-enthalpy function corresponding to temperature 16
-       \units J/kg
-       \type real
-  N30, \field Temperature 15
-       \note for Temperature-enthalpy function
-       \units C
-       \type real
-  N31, \field Enthalpy 15
-       \note for Temperature-enthalpy function corresponding to temperature 17
-       \units J/kg
-       \type real
-  N32, \field Temperature 16
-       \note for Temperature-enthalpy function
-       \units C
-       \type real
-  N33; \field Enthalpy 16
-       \note for Temperature-enthalpy function corresponding to temperature 16
-       \units J/kg
-       \type real
+  N20,N21,     N22,N23,     N24,N25,     N26,N27,     N28,N29,  \note fields as indicated
+  N30,N31,     N32,N33,     N34,N35,     N36,N37,     N38,N39,  \note fields as indicated
+  N40,N41,     N42,N43,     N44,N45,     N46,N47,     N48,N49,  \note fields as indicated
+  N50,N51,     N52,N53,     N54,N55,     N56,N57,     N58,N59,  \note fields as indicated
+  N60,N61,     N62,N63,     N64,N65,     N66,N67,     N68,N69,  \note fields as indicated
+  N70,N71,     N72,N73,     N74,N75,     N76,N77,     N78,N79,  \note fields as indicated
+  N80,N81,     N82,N83,     N84,N85,     N86,N87,     N88,N89,  \note fields as indicated
+  N90,N91,     N92,N93,     N94,N95,     N96,N97,     N98,N99,  \note fields as indicated
+  N100,N101,     N102,N103,     N104,N105,     N106,N107,     N108,N109,  \note fields as indicated
+  N110,N111,     N112,N113,     N114,N115,     N116,N117,     N118,N119,  \note fields as indicated
+  N120,N121,     N122,N123,     N124,N125,     N126,N127,     N128,N129,  \note fields as indicated
+  N130,N131,     N132,N133,     N134,N135,     N136,N137,     N138,N139,  \note fields as indicated
+  N140,N141,     N142,N143,     N144,N145,     N146,N147,     N148,N149,  \note fields as indicated
+  N150,N151,     N152,N153,     N154,N155,     N156,N157,     N158,N159,  \note fields as indicated
+  N160,N161,     N162,N163,     N164,N165,     N166,N167,     N168,N169,  \note fields as indicated
+  N170,N171,     N172,N173,     N174,N175,     N176,N177,     N178,N179,  \note fields as indicated
+  N180,N181,     N182,N183,     N184,N185,     N186,N187,     N188,N189,  \note fields as indicated
+  N190,N191,     N192,N193,     N194,N195,     N196,N197,     N198,N199,  \note fields as indicated
+  N200,N201;   \note fields as indicated
```

Comment:
I'm not super excited about this format.  I think I'm going to reformat them into individual lines with proper indenting.